### ISSUE TITLE
New queue_schema.rb file hardcodes Rails version

### ISSUE THREAD

**Issue by fractaledmind** [rails/solid_cache#211]
After #202, Solid Cache can't be added to any new Rails 7.2 applications:

```
bin/rails aborted!
ArgumentError: Unknown migration version "8.0"; expected one of "4.2", "5.0", "5.1", "5.2", "6.0", "6.1", "7.0", "7.1", "7.2" (ArgumentError)

          raise ArgumentError, "Unknown migration version #{version.inspect}; expected one of #{versions.sort.join(', ')}"
                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
/Users/fractaled/Code/Playgrounds/euruko-2024/db/cache_schema.rb:3:in `<main>'
```

I stumbled into this preparing my workshop for EuRuKo next week, where we are spinning up new Rails 7.2.1 apps and installing the Solid gems.

**Comment by fractaledmind** [comment]
cc @dhh and @djmb

**Comment by dhh** [comment]
Yeah, we can just set that to 7.2.

**Comment by dhh** [comment]
Need to fix that in SQ too.

**Comment by dhh** [comment]
Actually SQ is already set just to 7.1.

**Comment by dhh** [comment]
Fixed in 1.0.5.
